### PR TITLE
Use localized strings when creating functions

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
The templates already have these strings localized, so might as well use them.

Screen Shot 2019-03-19 at 11 02 52 AM

Screen Shot 2019-03-19 at 11 03 21 AM

### CODE REVIEW

File: src/templates/ScriptTemplateRetriever.ts [hunk 1]
```diff
@@ -77,15 +78,42 @@ export class ScriptTemplateRetriever extends TemplateRetriever {
...
+    const folder: string = path.join(templatesPath, 'resources');
+
+    try {
+        // Example: "en-US"
+        const parts: string[] = vscodeLang.split('-');
+        // Example: "en" for "english"
+        const language: string = parts[0];
+        // Example: "US" for "United States" (locale is optional)
+        // tslint:disable-next-line: strict-boolean-expressions
+        const locale: string | undefined = parts[1] || '[a-z]*';
```
Comment (StephenWeatherford) [inline src/templates/ScriptTemplateRetriever.ts]: Might be wise to allow uppercase here. I've frequently seen en-US for instance (https://tools.ietf.org/html/rfc5646).

Comment (StephenWeatherford) [reply src/templates/ScriptTemplateRetriever.ts]: Ah, I see your case-intensitive modifier below...

File: src/templates/ScriptTemplateRetriever.ts [hunk 2]
```diff
@@ -77,15 +78,42 @@ export class ScriptTemplateRetriever extends TemplateRetriever {
...
+
+        const files: string[] = await fse.readdir(folder);
+        const regExp: RegExp = new RegExp(`resources\\.${language}(-${locale})?\\.json`, 'i');
+        const matchingFile: string | undefined = files.find(f => regExp.test(f));
```
Comment (StephenWeatherford) [inline src/templates/ScriptTemplateRetriever.ts]: How are the resources done?  Do you always have specific locales, e.g. en-us resources, or can you have just "en" resources?  If the vscode language is "en", and you have en resources and en-uk resources, en should be preferred.

Comment (EricJizbaMSFT) [reply src/templates/ScriptTemplateRetriever.ts]: This is what the templates look like: https://github.com/Microsoft/vscode-azurefunctions/tree/master/resources/backupScriptTemplates/~2/resources

Not sure if that'll be very common, but fixed

**Pull request merged**